includes/hooks/template.php

```diff
@@ -316,10 +316,6 @@
 
 /**********************************
  *
- * Frontend notices display
+ * Frontend notices may display on any post, page or custom post type (used in Sensei blocks).
  */
-add_action( 'sensei_course_results_content_inside_before', array( $sensei->notices, 'maybe_print_notices' ) );
-add_action( 'sensei_single_course_content_inside_before', array( $sensei->notices, 'maybe_print_notices' ), 40 );
-add_action( 'sensei_single_lesson_content_inside_before', array( $sensei->notices, 'maybe_print_notices' ), 40 );
-add_action( 'sensei_taxonomy_module_content_inside_before', array( $sensei->notices, 'maybe_print_notices' ), 40 );
-add_action( 'sensei_single_quiz_content_inside_before', array( $sensei->notices, 'maybe_print_notices' ), 50 );
+add_action( 'init', array( $sensei->notices, 'maybe_print_notices' ) );
```

Comment:
Just an FYI that this changed caused a regression here: 
https://github.com/Automattic/sensei/issues/5562

We can't output content in `init` because headers might get set here.